src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -45,19 +54,59 @@ export class QuerySummary extends Component {
    */
   static options: IQuerySummaryOptions = {
     /**
-     * Specifies whether to display the search tips to the end user when there are no search results.
+     * Specifies whether to hide the information about the currently displayed range of results and only display the
+     * search tips instead.
+     *
+     * Default value is `false`.
+     */
+    onlyDisplaySearchTips: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+
+    /**
+     * Specifies whether to display the a message to the end user when there are no search results.
      *
      * Default value is `true`.
      */
-    enableSearchTips: ComponentOptions.buildBooleanOption({ defaultValue: true }),
+    enableNoResultsFoundMessage: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
+    // TODO : There is probably an other place where I will need to add this documentation for the query tag :<%-query%>)?
+    // TODO : I'm not sure if we should put an example in the description and if the description is precise enough?
     /**
-     * Specifies whether to hide the information about the currently displayed range of results and only display the
-     * search tips instead.
+     * Specifies a custom message to display when there are no search results.
      *
-     * Default value is `false`.
+     * A query tag ({@link QUERY_TAG}) can be added in the message. This tag will
+     * be replaced by the search box input who triggered the no results found page.
+     *
+     * Ex.:
+     *
+     * Searchbox input : "ThereIsNoResults"
+     *
+     * {@link QUERY_TAG} = <%-query%>
+     *
+     * Message input : "We are sorry, <%-query%> did not match any results."
+     *
+     * Final Message : "We are sorry, ThereIsNoResults did not match any results."
+     *
+     * Default value is `No results for <%-query%>`.
+     */
+    noResultsFoundMessage: ComponentOptions.buildStringOption({
+      defaultValue: DEFAULT_NO_RESULT_FOUND_MESSAGE,
+      depend: 'enableNoResultsFoundMessage',
+      postProcessing: (value, options) => value || DEFAULT_NO_RESULT_FOUND_MESSAGE
+    }),
+
+    /**
+     * Specifies whether to display the cancel last action link to the end user when there are no search results.
+     *
+     * Default value is `true`.
      */
-    onlyDisplaySearchTips: ComponentOptions.buildBooleanOption({ defaultValue: false })
```

Comment:
This variable still exists in the code it just have been moved for consistency with the layout of the options.